backend/picamera_backend.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+from time import time
+from typing import Dict
+
+from backend import Backend, PublishFunc
+from overlay import Canvas
+
+try:
```

Comment:
As mentioned before, this should be decided in the `Overlay` class that decides what `Backend` to choose from.